.gitignore

```
@@ -2,4 +2,6 @@
 *.log*
 lib
 logs
-node_modules
\ No newline at end of file
+node_modules
+test/studio/plugins/sanity-plugin-media-library
+.env
```

```suggestion
.env*
```

to prevent all sort of env files from being commited